content/articles/icann-60-day-lock-registrant-change.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: ICANN 60-Day Lock After Change of Registrant
+excerpt: After changing your registration info you cannot transfer your domain for sixty days.
+categories:
+- Contacts
+---
+
+# 60-Day Lock After Change of Registrant
+
+For all domains under ICANN management, all [gTLDs](/articles/what-is-tld/) and potentially some [ccTLDs](/articles/what-is-tld/), a new [ICANN policies that went into effect December 1st, 2016](https://www.icann.org/resources/pages/ownership-2013-05-03-en), making certain changes the registrant contact will lock the domain for 60 days. During that period, it will not be possible to transfer the domain to another registrar.
```

Comment:
```suggestion
For all domains under registries under contract with ICANN, including all [gTLDs](/articles/what-is-tld/), [newGTLDs](/articles/what-is-tld/), and potentially some [ccTLDs](/articles/what-is-tld/), a new [ICANN policies that went into effect December 1st, 2016](https://www.icann.org/resources/pages/ownership-2013-05-03-en), making certain changes the registrant contact will lock the domain for 60 days. During that period, it will not be possible to transfer the domain to another registrar.
```